rootdir/init.common.rc

```diff
@@ -161,6 +177,55 @@ on early-boot
     chown system system /sys/class/graphics/fb0/msm_fb_persist_mode
 
 on boot
+    # WLAN and BT MAC
+    chown system system /sys/devices/soc/soc:bcmdhd_wlan/macaddr
```

Comment:
@humberos I don't think that this is common for Yoshino platform too

Reply:
I think it will not affect boot process.
We will get only warnings 
anyway thanks for the hint

Reply:
please leave this on loire and tone (bcm platforms)

Reply:
Done!